src/llm_benchmark/adapters/llms/litellm_adapter.py

```
@@ -93,23 +114,40 @@ def complete(self, request: LLMRequest) -> LLMResponse:
+
+                return LLMResponse(
+                    text=raw_response.choices[0].message.content,
+                    input_tokens=raw_response.usage.prompt_tokens,
+                    output_tokens=raw_response.usage.completion_tokens,
+                    latency=latency,
+                    raw=raw_response.model_dump(),
+                )
+            except RateLimitError as exc:
+                last_exc = exc
+                if attempt < _MAX_RETRIES - 1:
+                    jitter = backoff * random.uniform(-0.1, 0.1)
```

Ruff (0.15.10)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)